app/src/main/java/org/mozilla/tv/firefox/webrender/WebRenderFragment.kt

```diff
@@ -284,6 +289,8 @@ class WebRenderFragment : EngineViewLifecycleFragment(), Session.Observer {
         if (session.url == APP_URL_HOME) {
             browserOverlay?.visibility = View.VISIBLE
         }
+
+        webView.addJavascriptInterface(YouTubeBackHandler(), "youtubeBackHandler")
```

Comment:
`youtubeBackHandler` gets written directly into the DOM so you probably want to name it something:
- identifiable by website owners wondering what this random object is
- identifiable by us to know we added it
- that won't get clobbered by site content

[For media session, we use `_firefoxTV_playbackStateObserverJava`](https://github.com/mozilla-mobile/firefox-tv/blob/23a1288990e22a3fc06a2975bac0c307775a9e8d/app/src/main/java/org/mozilla/tv/firefox/webrender/VideoVoiceCommandMediaSession.kt#L52).

By the way, see `VideoVoiceCommandMediaSession` for an example of how you can break a JavascriptInterface dependent class into a separate file.